### DISCUSSION TITLE
config error data not in correct format

### DISCUSSION THREAD

**Discussion by whateverx28** [code4recovery/12-step-meeting-list#1341] [Q&A] [answered]
![Meeting List ](https://github.com/code4recovery/12-step-meeting-list/assets/158858278/7622f31e-0079-4919-8f88-6800c5ad23ab)
Hello, I followed directions to the letter but do not see the config error that data is not in correct format. URL https://aarichmond2stg.wpenginepowered.com/meetings/?meeting / I entered these two meetings manually. The good news is that I see the maps and they are accurate. Thanks for your help. I am new to this.

**Comment by joshreisner** [comment]
hi @whateverx28 is the problem here that these meetings appear as drafts and you want them to show up on the site?

could you send a screenshot of the edit meeting page for one of these meetings?

thanks

**Reply by joshreisner** [reply]
hi @whateverx28 ! i think i may have steered you wrong when i said "send a screenshot" - because email attachments don't come through on github unfortunately. could you come to the topic and upload them?

for what it's worth, firefox has a nice built-in full-page capture button.

**Reply by joshreisner** [reply]
here's sort of a minimum-viable in-person meeting. i put arrows pointing to the address and "can i attend this meeting in-person" since that's what sometimes trips people up - not sure if that's what's happening in your case though

![Screenshot 2024-02-07 at 10-54-13 Edit Meeting “Test Meeting” ‹ Code for Recovery — WordPress](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/af84d45d-5056-479a-b849-342cda2b63dc)

**Comment by whateverx28** [comment]
Josh,

Thank you so much for this. Attached four screen shots that show top to bottom of the edit meeting page for one of the two meetings I entered manually.  I think there has to be a better way to show you the whole page, but I do not know it. If you do, please tell me. I think it’s too hard to read what I did entered the meeting. The good news is that the map shows up exactly right.

Karen

**Comment by whateverx28** [comment]
Josh,

 

I am going to uninstall and reinstall because I inherited this app. Thank you for responding. Back to you if I have the same problem after the procedure.


Thank you very much

Karen

**Comment by whateverx28** [comment]
Josh, 

 

I deactivated the 12 Step Meeting List because I think uninstalling and reinstalling is the best way to go. I inherited this site including the plugins, which is why I want to uninstall and reinstall. 

 

But the app won’t delete. It acts like it’s going to delete but it does not. Do you have any suggestions? 

 

Thank you!

Karen

**Reply by joshreisner** [reply]
i don't think deactivating / removing the plugin would help, that's typically not a step we recommend.

that said, i tried to do that on our dev site and i do see an error, so sounds like that could be an unrelated bug.

<img width="806" alt="Screenshot 2024-02-07 at 5 01 51 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/5a45d7ab-7f08-4a1b-afd4-81dc4e26de59">

at any rate, i would reactivate the plugin and just focus on getting one of those meetings live. if you can upload a screenshot here that would help, or just summarize what it is you're trying to add? what's the meeting name / types / online meeting info / address / etc?

**Comment by whateverx28** [comment]
Joah, 

 

OK I activated the plugin. I understand and thanks for the info.

 

Attached are five screen shots of one live meeting I entered manually, one showing the front end, and four of the back end. When I preview this entry, I get a “Configuration error: data is not in the correct format.”

 

The map function works very well. There are six districts in the greater Richmond VA area, though I haven’t specificed a district.

 

I hope these are not hard for you to read. If you know a better way I can send this info, please tell me.

 

Thank you –

Karen

**Reply by joshreisner** [reply]
can you upload the screenshots? we do not see them. thanks

**Reply by whateverx28** [reply]
![Bottom section Edit mtg 4 of 4](https://github.com/code4recovery/12-step-meeting-list/assets/158858278/6cb16b85-e92d-411c-90e5-bf17980a41ea)
![Meetings -- Draft Front End](https://github.com/code4recovery/12-step-meeting-list/assets/158858278/08fab367-1921-445a-b9f6-605d8e5295ef)
![Second section Edit mtg 2 of 4](https://github.com/code4recovery/12-step-meeting-list/assets/158858278/d7d3870f-54a1-49ef-abc3-5eae189b9851)
![Third section Edit mtg 3 of 4](https://github.com/code4recovery/12-step-meeting-list/assets/158858278/f7976c58-7fa9-46f8-ac50-aab4c34655b8)
![Upper section Edit mtg 1 of 4](https://github.com/code4recovery/12-step-meeting-list/assets/158858278/32eeac7a-31ad-41ba-b174-cfc534135932)

Karen M.

**Reply by joshreisner** [reply]
this looks perfect, thanks! if you hit the "Publish" button does it publish?

**Comment by whateverx28** [comment]
When I hit publish, I see the same error. Configuration error: data is not in the correct format. I am studying this because something else is wrong; at least I know it’s not my manual entries. Thank you for the clarification.

**Reply by joshreisner** [reply]
so the status is still "Draft" even after clicking Publish?

i think the message you are seeing is because the feed is empty, which i think is because there aren't published meetings. so i think the priority is to publish one of these meetings

i wonder if you could create a new test meeting at a new location and see if that works?

**Comment by whateverx28** [comment]
I don’t see the status when I hit Publish from the Messages page. It just shows up as configuration error. The two meetings I entered manually are published meetings, that is, they are bona fide meetings in the Richmond VA area. 

 

I don’t know what you mean by creating a new test meeting at a new location, but I think I will create a new WordPress instance and start over again. I inherited these plugins, that is, I imported “wp-content” into “wpeprivate” in WordPress in the wp-engine host.  

 

Thank you very much for your help.

**Reply by joshreisner** [reply, marked as answer]
![image](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/58e8e4e8-187f-4870-830e-7d9951b44fb2)

in this screenshot, above the blue Publish button, it says "Status: Draft" - it won't display on the frontend until it says "Status: Published"

**Comment by whateverx28** [comment]
Oh, I get it. Thank you. I will go in again and fix that.

**Comment by whateverx28** [comment]
Josh, that worked!! I am so happy. I can see the two meetings! Wow! Thank you so much.

Another problem which has nothing to do with the 12 Step Mtg app is a plugin called Elementor, again I inherited it. It gave me another error when I tried to see the meetings. I switched back to simple WordPress. I am killing that Elementor plugin for sure. 

You are the best!!
